en/middleware/xrce_dds.md

```diff
@@ -306,6 +344,59 @@ ROS 2 uses the following QoS settings (by default) for publishers and subscripti
 Deadline, lifespan, and lease durations are also all set to "default".
 <!-- From https://github.com/PX4/PX4-user_guide/pull/2259#discussion_r1099788316 -->
 
+
+## DDS Topics YAML
```

Comment:
@beniaminopozzan OK, I moved both headings up one level, and added this heading here for DDS Topics YAML. I have linked to this section (which duplicates your original text) from above and from the ROS2 Guide). We might even have more information/links here, but this is enough for now - i.e. in some places we talk a little about the file format where we could redirect here.

Before I did all that I rebased on main, so this now includes all your other PRs.

I'm going to leave with you to think about this as I have to go catch a plane. If you want to do a further round of edits let me know, otherwise we can merge and add next bits as a post-process. UP to you. I'll look in to see if you have commented in a couple of hours.